### PR TITLE
New api

### PR DESCRIPTION
**Pull request by roszpun**

### CODE REVIEW

File: src/redstone-api-adapter.js [hunk 1]
```diff
@@ -41,53 +112,57 @@ export default {
...
-        .query()
-        .symbol(symbol)
-        .forLastDays(daysCount)
-        .exec({ defaultProvider: provider });
+      return await getApiData(symbol, { provider, daysCount });
+      // return await redstone
+      //   .query()
+      //   .symbol(symbol)
+      //   .forLastDays(daysCount)
+      //   .exec({ defaultProvider: provider });
```
Comment (cehali) [inline src/redstone-api-adapter.js:165]: Maybe remove it?

File: src/redstone-api-adapter.js [hunk 2]
```diff
@@ -7,6 +7,70 @@ const COINGECKO_URL = "https://dy9r79395zjk.cloudfront.net/api/v3";
...
+    if (value !== undefined) {
+      url.searchParams.append(key, value);
+    }
+  });
+  const result = await axios.get(url.toString());
+  return result.data;
+};
+
+const getApiData = async function (symbol, opts) {
+  const baseUrl = "http://localhost:9000/prices";
```
Comment (cehali) [inline src/redstone-api-adapter.js:40]: Is it going to work with `localhost`?

Comment (roszpun) [reply src/redstone-api-adapter.js:40]: No, my mistake

File: src/redstone-api-adapter.js (hunk 1 above)
Comment (roszpun) [reply src/redstone-api-adapter.js]: Removed

**Pull request merged**